src/raw/tests.rs

```diff
@@ -461,6 +467,209 @@ test_range! {
     "a", "b", "c", "d", "e", "f"
 }
 
+test_range! {
+    fst_range_20,
+    min: Bound::Included(vec![b'a', b'a', b'a']), max: Bound::Unbounded,
+    imin: 1, imax: 4,
+    "a", "aaa", "aba", "aca"
+}
+
+test_range! {
+    fst_range_21,
+    min: Bound::Included(b"aab".to_vec()), max: Bound::Unbounded,
+    imin: 2, imax: 4,
+    "a", "aaa", "aba", "aca"
+}
+
+test_range! {
+    fst_range_22,
+    min: Bound::Excluded(b"aab".to_vec()), max: Bound::Unbounded,
+    imin: 2, imax: 4,
+    "a", "aaa", "aba", "aca"
+}
```

Comment:
If one of them has an inclusive min-bound and the other one has an exclusive min-bound, should they not have different results?

Reply:
No because:

a
aaa
-- aab
aba
aca

No, because the bound is in between two of the values.